wait-for-idle-runner/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Wait for idle runner GitHub Action
+
+An action that waits for an idle runner.
+It returns "::set-output name=result::true" when the runner is idle.
```

Comment:
I'd maybe say something like this instead, as it doesn't really _return_ that text

```suggestion
The `result` output is set to reflect the state of the runner.
```

Feel free to change that to better match what the `result` output really is.